stencil-library/src/api/Api.ts

```diff
@@ -23,7 +23,7 @@ export interface IApiResponseCollection<T> extends IApiResponse<T> {
 }
 
 const Api = (authToken: string) => {
-  const apiOrigin = 'http://localhost:3000';
+  const apiOrigin = 'https://justifi.ai';
```

Comment:
This shouldn't be coming from an env file?

Reply:
Good question, I just left it as it was.